### DISCUSSION TITLE
Legend displayed in TerriaMap is Tiny.

### DISCUSSION THREAD

**Discussion by lstillwe** [TerriaJS/terriajs#5510] [Q&A]
When I add a png to a raster style in GeoServer (Size 700 x 150 pixels), and use WMS GetCapabilties in the catalog item, it display the legend, but the image is tiny. How do I size this image to get it to properly display in the legend area of the data panel. 
![Screen Shot 2021-05-25 at 1 56 06 PM](https://user-images.githubusercontent.com/1794116/119545583-fc293080-bd60-11eb-86d5-552d23825321.png)

**Comment by malnajdi** [comment]
If you are using Geoserver as your source of data you can check this url. https://docs.geoserver.org/latest/en/user/services/wms/get_legend_graphic/index.html

What i have done in my project i had to edit https://github.com/TerriaJS/terriajs/blob/master/lib/ReactViews/Workbench/Controls/Legend.jsx#L41 I added a new line which is 
```
legendUrl.url = legendUrl.url.replace("LEGEND_OPTIONS=", "LEGEND_OPTIONS=fontName:Dejavo;");
```
